src/gui/mainwindow.ui

```diff
@@ -285,6 +286,17 @@
     <string>Display Top Toolbar</string>
    </property>
   </action>
+  <action name="actionShow_Status_bar">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Status bar</string>
```

Comment:
"Status bar" -> "Status **B**ar"

>&amp; missing?

You can give one if you want.

And screenshot the latest git version, not an outdated old version.